Спринт 14. ShareIt. Добавление контроллеров для пользователей и вещей.

Skok-off opened the pull request.

Skok-off committed:
feature: Спринт 14. ShareIt

jeggga commented on src/main/java/ru/practicum/shareit/helper/ColorsForConsole.java:
```diff
@@ -0,0 +1,11 @@
+package ru.practicum.shareit.helper;
+
+public class ColorsForConsole {
+    public static final String ANSI_RESET = "\u001B[0m";
```
Лучше переписать через Enum

jeggga commented on src/main/java/ru/practicum/shareit/item/controller/ItemController.java:
```diff
@@ -0,0 +1,62 @@
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public ItemDto create(@RequestBody @Valid ItemDto itemDto,
+                          @RequestHeader("X-Sharer-User-Id") Integer owner) {
```
X-Sharer-User-Id - много, где используется, следует вынести в константу

Skok-off committed:
refactor: Спринт 14. Исправления после ревью.

-ColorsForConsole переделал в Enum
-X-Sharer-User-Id вынес в константу
-поправил вывод логов в хэндлере на параметризованные, вместо сложения строк (совет от IDEA)
